packages/stack/src/views/Stack/CardContainer.tsx

```diff
@@ -128,6 +130,38 @@ function CardContainer({
 
   const { colors } = useTheme();
 
+  const [pointerEvents, setPointerEvents] = React.useState<'box-none' | 'none'>(
+    'box-none'
+  );
+
+  const valueListenerCallback = React.useCallback(
```

Comment:
You can also move the callback to inside `useEffect`. It'll remove the need for adding `useCallback`.